typescript/src/detectors/typescript-do-not-forward-user-ip-address-using-http-proxy/typescript-do-not-forward-user-ip-address-using-http-proxy_non-compliant.ts

```diff
@@ -0,0 +1,41 @@
+// Copyright Amazon.com, Inc. or its affiliates. All Rights Reserved.
+// SPDX-License-Identifier: MIT-2.0
+
+// {fact rule=typescript-do-not-forward-user-ip-address-using-http-proxy@v1.0 defects=1}
+import express from 'express';
+import { createProxyMiddleware, RequestHandler } from 'http-proxy-middleware';
+import { Request, Response } from 'express';
+
+const app = express();
+
+const nonCompliant = (): void => {
+    const app = express();
+
+    const onProxyReq = (proxyRes: Request): void => {
+        console.log('req');
+    };
+
+    const onProxyRes = (proxyRes: Response): void => {
+        console.log('res');
+    };
+
+    const getProxy = (): RequestHandler => {
+        return createProxyMiddleware({
+            target: "API_SERVICE_URL",
+            changeOrigin: true,
+            onProxyReq,
+            onProxyRes,
+            // Noncompliant: xfwd's defaults value is true.
```

Comment:
what is the default value of the `xfwd`, since you mention above that the xfwd default value as false